### PR TITLE
fix(curriculum): add code example in step 12 of Cat Photo App project

### LINKED ISSUE #51258: Provide a code example for Step 12 of the cat photo project
### Describe the Issue

I know that [step 12 ](https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-12)of the cat photo app has been rewritten a few times, but it still stumps a lot of beginners on the [forum](https://forum.freecodecamp.org/search?expanded=true&q=Learn%20HTML%20by%20Building%20a%20Cat%20Photo%20App%20-%20Step%2012%20after%3A2023-06-01). 
I think we should add a code example of how to nest an anchor element in a paragraph element. 

## Proposed change 
Please read through the updated change in this comment here
https://github.com/freeCodeCamp/freeCodeCamp/issues/51258#issuecomment-1693784945

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-12

### PR DESCRIPTION
**Pull request by samkim777**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #51258

### CODE REVIEW

**Comment by jdwilkin4** [comment]
@samkim777 

Please make sure to properly link the issue to this PR 
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md [hunk 1]
```diff
@@ -11,6 +11,14 @@ In the previous step you turned the words `link to cat pictures` into a link by
 
 In the text of your `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com` by adding opening and closing anchor (`a`) tags around these words.
```
Comment (jdwilkin4) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md:12]: You did add in the new instructions like the ticket mentioned.
However, you also kept the old instructions too so now the entire group of instructions doesn't make sense

It currently read like this:

```

In the previous step you turned the words link to cat pictures into a link by placing them between opening and closing anchor (a) tags. You can do the same to words inside of an element, such as a p element.

In the text of your p element, turn the words cat photos into a link to https://freecatphotoapp.com by adding opening and closing anchor (a) tags around these words.

You can turn any text into a link, such as the text inside of a p element.

<p>I think <a href="https://www.freecodecamp.org/">freeCodeCamp</a> is great.</p>
In the text of your p element, turn the words cat photos into a link by adding opening and closing anchor (a) tags around these words. Then set the href attribute to https://freecatphotoapp.com/
```

You will need to fix this so we only see the new instructions and there are no duplicate sentences

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md (hunk 1 above)
Comment (samkim777) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md]: Whoops my bad! I think I fixed the new instructions now. 
I'm not sure why some of the checks are no longer passing though...  :(

Comment (jdwilkin4) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md]: I am not sure why the e2e tests are failing.
maybe one of the other reviewers can spot why.
I tried looking into the logs but couldn't find any helpful error messages

**Review by jdwilkin4 — APPROVED** [review]
The updates in the instructions look good to me

**Comment by gikf** [comment]
Hey @samkim777

CI errors seem to be unrelated and originating from the `main` at the time when PR branch was created. Could you update the PR branch with changes from the `freeCodeCamp:main`?

Let us know if you'd need help with that.

Happy contributing.

**Comment by samkim777** [comment]
Hello @gikf!
I think I updated the PR branch correctly. Could you take a look?

Thanks!

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Comment by gikf** [comment]
Hi @samkim777. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**